archaius2-core/src/main/java/com/netflix/archaius/config/AbstractConfig.java

```diff
@@ -143,8 +142,7 @@ public String getString(String key) {
         if (value instanceof String) {
             return interpolator.create(getLookup()).resolve(value.toString());
         } else {
-            throw new UnsupportedOperationException(
-                    "Property values other than String not supported");
+            return value.toString();
```

Comment:
Getters for specific types continue to work as before.  This just ensures that you can get a string representation of a property that is not stored as a string.